gbcamextract.c

```diff
@@ -1,352 +1,245 @@
 /*
  * gbcamextract.c, extracts photos from Game Boy Camera save files.
- * 
+ *
```

Comment:
Vim automatically removes trailing whitespace. I wasn't trying to mess with the license. If it's a problem, I can undo.